qiita_pet/templates/private_studies.html

```diff
@@ -3,13 +3,18 @@
 <link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
 
 <script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
 
 <script type="text/javascript">
 var current_study;
 
 $(document).ready(function() {
-    $('#user-studies-table').dataTable({"order": [[ 1, "asc" ]]});
-    $('#shared-studies-table').dataTable({"order": [[ 1, "asc" ]]});
+        $('#user-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
+        $('#shared-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
```

Comment:
No... and I can't test locally as my demo database is hosed

On Thu, Feb 19, 2015 at 1:14 PM, adamrp <notifications@github.com> wrote:

> In qiita_pet/templates/private_studies.html
> <https://github.com/biocore/qiita/pull/862#discussion_r25021020>:
>
> >
> >  <script type="text/javascript">
> >  var current_study;
> >
> >  $(document).ready(function() {
> > -    $('#user-studies-table').dataTable({"order": [[ 1, "asc" ]]});
> > -    $('#shared-studies-table').dataTable({"order": [[ 1, "asc" ]]});
> > +        $('#user-studies-table').dataTable({
> > +            order: [[ 1, "asc" ]],
> > +            columnDefs: 'natural'});
> > +        $('#shared-studies-table').dataTable({
> > +            order: [[ 1, "asc" ]],
> > +            columnDefs: 'natural'});
>
> Even when I change the code to reflect the example, it doesn't work.
> There's also not great documentation on this plugin, which doesn't help...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/862/files#r25021020>.
>